build_env.sh

```diff
@@ -3,6 +3,7 @@
 cd /workspaces/build
 git clone https://github.com/emscripten-core/emsdk.git .emsdk
 cd .emsdk
+git checkout 3.1.54
```

Comment:
It's probably safer to run the following commands with `3.1.54` rather than checking out an old tag. I'm not sure how emscripten installs and don't know if it'll lead to package conflicts

Reply:
I'm running a build right now, and it looks like the cores are building ok (no crashes yet!).

In this environment, I've added your repo, used apt-get install emscripten, and then checked out that earlier version of emsdk - seems to be working ok.

Reply:
You shouldn’t need to do any checkouts if you install from my repo?